pennylane/tape/qnode.py

```diff
@@ -865,6 +884,7 @@ def qfunc_decorator(func):
             interface=interface,
             diff_method=diff_method,
             mutable=mutable,
+            max_expansion=max_expansion,

```

Comment:
I know this has already been code reviewed, but for my understanding - we have added the `max_expansion` attribute but have we implemented the effects of changing `max_expansion`?

Reply:
Yep! The change is on line 781, `max_expansion=10`.